projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/answers/AnswerSummary.java

```diff
@@ -0,0 +1,84 @@
+package org.batfish.datamodel.answers;
+
+import com.google.common.base.Strings;
+
+public class AnswerSummary {
```

Comment:
I guess it in fact is, using the default Jackson inspection of objects. That's not the usual standard in Batfish, but ok with me.